File: Go/File/remove-a-file.go
package main

import (
	"log"
	"os"
)

func main() {
	// Remove a file
	removeFile("assets/remove/README.md")
}

// Remove a file from the file system
func removeFile(path string) {
	err := os.Remove(path)
	if err != nil {
		log.Fatalln(err)
	}
}

File: Dart/File/remove_a_file.dart
import "dart:core";
import "dart:io";

void main() {
  // Remove a file.
  removeAFile("assets/remove/zVA3m3BEJ5wSe45v93Fi8788z4GoKjnW/README.md");
}

// Remove a file in a specefied path.
void removeAFile(final String path) {
  File(path).deleteSync();
}
